sct.py

```diff
@@ -697,7 +697,11 @@ def list_resources_on_azure():
               type=click.Choice(AwsArchType.__args__),
               default='x86_64',
               help="architecture of the AMI (default: x86_64)")
-def list_images(cloud_provider: str, branch: str, version: str, regions: List[str], arch: AwsArchType):
+@click.option('-o', '--output-format',
+              type=str,
+              default='table',
+              help="")
```

Comment:
let's add help, cause it's not clear what text means

why no have json output ?, so we can clearly read specific field ?

Reply:
changed to json format